addons/resource.language.en_gb/resources/strings.po

```diff
@@ -2860,59 +2869,59 @@ msgstr ""
 
 #: xbmc/settings/AdvancedSettings.cpp
 msgctxt "#669"
-msgid "Verbose logging for SMB library"
+msgid "Verbose logging for \"SMB\" library"
```

Comment:
Why is this in quotes? I'm horrible at grammar, but I'm pretty sure there shouldn't be quotes on SMB. http://grammarist.com/grammar/quotation-marks/